hardhat.config.ts

```diff
@@ -53,13 +53,12 @@ const config: HardhatUserConfig = {
         blockNumber: blockNumber.forkAt,
       },
     },
-
-    avax_mainnet: {
+    avalanche: {
```

Comment:
for most of us I think it is much easier to remember

<chain_name>_<chain type> especially that we are going to have more chains

ETH has multiple testnet, but people are usually using 1

Can we revert this pls

Reply:
As I wrote in the commit message, then I can't use Defender Admit Client so all changes regarding deployment script to work on both test and production are useless. And for me, I'm comfortable with using chain's own name and I think it's more straightforward than using our own convention.